core/src/link_resolver.rs

```diff
@@ -192,9 +176,7 @@ impl LinkResolverTrait for LinkResolver {
         )
         .await?;
 
-        // FIXME: Having an env variable here is a problem for consensus.
-        // Index Nodes should not disagree on whether the file should be read.
-        let max_file_size: Option<u64> = read_u64_from_env(MAX_IPFS_FILE_SIZE_VAR);
+        let max_file_size = ENV_VARS.max_ipfs_file_bytes().map(|n| n as u64);
```

Comment:
Will we have the same `panic` as before with `read_u64_from_env`? I imagine the crate you're using already handles this when trying to parse from the env var right?

Because it will do `u64::from_str` the same way.

Reply:
Yeah, we'd panic.